src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java

```diff
@@ -1460,14 +1460,38 @@ private void run2() {
 
         // --- Stargates
         if (Loader.isModLoaded("GalaxySpace") && Loader.isModLoaded("SGCraft"))
-            for (int i = 0; i < 6; i++)
+            for (byte i = 0; i < 6; ++i)
```

Comment:
i++ :
    create a temporary copy of i
    increment i
    return the temporary copy

++i :
    increment i
    return i

Reply:
Bytecode generated is identical in both cases!

Reply:
Then this is pointless.